master/custom/factories.py

```diff
@@ -224,8 +224,8 @@ class UnixAsanBuild(UnixBuild):
     buildersuffix = ".asan"
     configureFlags = ["--without-pymalloc", "--with-address-sanitizer"]
     factory_tags = ["asan", "sanitizer"]
-    compile_environ = {'ASAN_OPTIONS': 'detect_leaks=0:allocator_may_return_null=1'}
-    test_environ = {'ASAN_OPTIONS': 'detect_leaks=0:allocator_may_return_null=1'}
+    compile_environ = {'ASAN_OPTIONS': 'detect_leaks=0:allocator_may_return_null=1:handle_segv=0'}
```

Comment:
Can you please add a comment mentioning https://bugs.python.org/issue42985 to explain why SIGSEGV is ignored on purpose?

Reply:
Done